assets/js/blocks/cart-checkout/cart-i2/frontend.js

```diff
@@ -5,25 +5,27 @@ import {
 	withStoreCartApiHydration,
```

Comment:
We can turn this file to TS, just not now. after merging this PR, to keep a linear history in case we want to check what changed in the future.